Automattic/wp-calypso
# Replace client/languages with packages/languages

Language data currently held in client/languages is required outside the calypso scope.

We need this data to enable moving the `localizeUrl` function outside of calypso (I'm looking to move this function into i18n-calypso.)

For examples, linking to support documentation in Gutenboarding: #45610/#46670, or linking to the help/contact page in the [editing toolkit](https://github.com/Automattic/wp-calypso/blob/ec22864/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx#L165).

#### Changes proposed in this Pull Request

* Replaces `client/languages` with `packages/languages`.

#### Testing instructions

Unit tests check basic data is being loaded:
* `yarn workspace @automattic/languages test`

Manually you can:
* Alter the language-meta.json file (e.g. change a locale name)
* Check the resulting change is seen in language pickers: e.g. http://calypso.localhost:3000/new/language-modal
